src/common/modal/MuiModal.jsx

```diff
@@ -0,0 +1,30 @@
+import { Modal, Box } from "@mui/material";
+
+const style = {
+  position: "absolute",
+  top: "50%",
+  left: "50%",
+  transform: "translate(-50%, -50%)",
+  width: 400,
+  bgcolor: "background.paper",
+  borderRadius: 2,
+  boxShadow: 24,
+  border: 'none',
+  outline: 'none',
+  px: 4,
+  py: 2,
+};
+
+const MuiModal = ({ component, children, open, handleClose }) => {
```

Comment:
Let's not use any reserved/third-party keyword.

Reply:
removed